Makefile

```diff
@@ -356,6 +356,22 @@ packs-tests: requirements .packs-tests
 	@echo
 	. $(VIRTUALENV_DIR)/bin/activate; find ${ROOT_DIR}/contrib/* -maxdepth 0 -type d -print0 | xargs -0 -I FILENAME ./st2common/bin/st2-run-pack-tests -x -p FILENAME
 
+
+REPOSRC := https://github.com/StackStorm/st2tests.git
+LOCALREPO := st2tests_repo
+TESTDIR := docs mistral chatopsCI
```

Comment:
Add a new directory for test suites here. RF will generate reports/logs for the test suites in the respective directory:
For Example: For docs:
```
Output:  /home/vagrant/git/st2/st2tests/st2tests_repo/robotfm_tests/docs/output.xml
Log:     /home/vagrant/git/st2/st2tests/st2tests_repo/robotfm_tests/docs/log.html
Report:  /home/vagrant/git/st2/st2tests/st2tests_repo/robotfm_tests/docs/report.html
```